experimental/rsocket-src/RSocketRequester.cpp

```diff
@@ -41,31 +43,29 @@ std::shared_ptr<Subscriber<Payload>> RSocketRequester::requestChannel(
   return standardReactiveSocket_->requestChannel(std::move(responseSink));
 }
 
-void RSocketRequester::requestStream(
-    Payload request,
-    std::shared_ptr<Subscriber<Payload>> responseSink) {
-  eventBase_.runInEventBaseThread(
-      [ this, request = std::move(request), responseSink ]() mutable {
-        standardReactiveSocket_->requestStream(
-            std::move(request), std::move(responseSink));
-      });
-}
+yarpl::Reference<yarpl::Flowable<Payload>> RSocketRequester::requestStream(
+    Payload request) {
+  auto flowable = yarpl::Flowable<Payload>::create([total = 0](
+      yarpl::Subscriber<Payload> & subscriber, int64_t requested) mutable {
+    subscriber.onNext(Payload());
+    return std::make_tuple(int64_t{1}, false);
+  });
+  return flowable;
 
-std::shared_ptr<yarpl::flowable::Flowable<reactivesocket::Payload>>
-RSocketRequester::requestStream(reactivesocket::Payload request) {
-  auto& eb = eventBase_;
-  auto srs = standardReactiveSocket_;
-  return yarpl::flowable::Flowable<reactivesocket::Payload>::create(
```

Comment:
Here since the underlying RSocket code just takes a `Subscriber` and does the `Subscriber`/`Subscription` behavior itself, I just need to pass the `Subscriber` into it ... after awkwardly scheduling the subscribe onto an eventBase. 

This code effectively:

```cpp
    auto& eb = eventBase_;
    auto srs = standardReactiveSocket_;
    return yarpl::Flowable<Payload>::create(
        [&eb, request = std::move(request), srs = std::move(srs) ](
            auto uptr_subscriber) mutable {
          auto os =
              std::make_shared<OldToNewSubscriber>(std::move(uptr_subscriber));
          eb.runInEventBaseThread([
            request = std::move(request),
            os = std::move(os),
            srs = std::move(srs)
          ]() mutable { srs->requestStream(std::move(request), std::move(os));
          });
        });
```

How do you recommend doing this with the new `Flowable`?

In short, I don't think the `create(Emitter)` can do this one as that assumes synchronous execution, whereas this is an async use case.